branca/utilities.py

```diff
@@ -162,10 +163,20 @@ def color_brewer(color_code, n=6):
                              ' and ' + str(max(matching_quals))
                              )
         else:
+            longest_scheme_name = base_code
+            longest_scheme_n = 0
+            for sn_name in schemes.keys():
+                if '_' not in sn_name:
+                    continue
+                if sn_name.split('_')[0] != base_code:
+                    continue
+                if int(sn_name.split('_')[1]) > longest_scheme_n:
+                    longest_scheme_name = sn_name
```

Comment:
shouldn't we update `longest_scheme_n` here as well?

Reply:
Nice catch!
It's fixed